src/main/java/com/example/crudapi/entity/Nationality.java

```diff
@@ -0,0 +1,35 @@
+package com.example.crudapi.entity;
+
+public class Nationality {
+
+    private int nameId, countryId;
```

Comment:
このようにクラスをフィールドに持たせる作りにしても良いかな。
```suggestion
    private Name name;
```